cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -126,7 +127,10 @@ func StartTestServer(t Logger, instanceOptions *TestServerInstanceOptions, custo
 	if !ok {
 		return result, fmt.Errorf("failed to get current file")
 	}
-	s.SecureServing.ServerCert.FixtureDirectory = path.Join(path.Dir(thisFile), "testdata")
+	s.SecureServing.ServerCert.FixtureDirectory = filepath.Join(path.Dir(thisFile), "testdata")
```

Comment:
same approach exists in `staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/testing/testserver.go`

is there a cleaner approach that would work for go and bazel test envs?